static/app/views/alerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -65,17 +65,15 @@ class RuleNode extends React.Component<Props> {
     let initialVal;
     if (data) {
       if (data[name] === undefined && !!fieldConfig.choices.length) {
-        if (fieldConfig.initial) {
-          initialVal = fieldConfig.initial;
-        } else {
-          initialVal = fieldConfig.choices[0][0];
```

Comment:
Even though the field when changed will provide strings (as per https://github.com/getsentry/sentry/pull/28828), I believe this original intention behind this code was to always have a value in the field even on first load with no default set.

This coerces _every_ possible value into a string so that those fields will start of populated.